File: 1165 - Numero Primo/numero_primo.py
n = int(input())
for _ in range(n):
    x = int(input())
    primo = x > 1
    i = 2
    while i * i <= x:
        if x % i == 0:
            primo = False
            break
        i += 1
    if primo:
        print(f"{x} eh primo")
    else:
        print(f"{x} nao eh primo")

File: 1165 - Numero Primo/numero_primo.c
#include <stdio.h>

int main(){
    int n;
    scanf("%d", &n);
    while(n--){
        long long x;
        scanf("%lld", &x);
        int primo = x > 1;
        for(long long i = 2; i * i <= x; i++){
            if(x % i == 0){ primo = 0; break; }
        }
        if(primo) printf("%lld eh primo\n", x);
        else printf("%lld nao eh primo\n", x);
    }
    return 0;
}
